source/extensions/filters/http/oauth2/oauth.h

```diff
@@ -20,9 +22,10 @@ class FilterCallbacks {
   virtual ~FilterCallbacks() = default;
 
   virtual void onGetAccessTokenSuccess(const std::string& access_token,
-                                       std::chrono::seconds expires_in) PURE;
+                                       std::chrono::seconds expires_in, const std::string& subject,
+                                       const std::vector<std::string>& groups) PURE;
 
-  virtual void sendUnauthorizedResponse() PURE;
+  virtual void sendUnauthorizedResponse(const absl::string_view& message) PURE;
```

Comment:
`absl::string_view` can be passed by value

> Unlike other string types, you should pass string_view by value just like you would an int or a double because string_view is a small value.

(from https://abseil.io/tips/1)

Reply:
I was going to make the same comment, but there are a few cases here and there:

```
$ git grep 'const absl::string_view&' -- '*.h'
source/common/common/utility.h:  static std::string removeCharacters(const absl::string_view& str,
source/common/http/utility.h:void extractHostPathFromUri(const absl::string_view& uri, absl::string_view& host,
source/common/http/utility.h:std::string localPathFromFilePath(const absl::string_view& file_path);
```

so I refrained myself 😄